src/utils.ts

```diff
@@ -95,7 +96,7 @@ export async function sendFile(
   headers.set(
     "Content-Type",
     headers.get("Content-Type") ||
-      (contentType(pathFile.replace("/", "\\")) || "application/octet-stream"),
+      (contentType(path.replace("/", "\\")) || "application/octet-stream"),
```

Comment:
this point if accessing the root folder or accessing the default index then it is undefined. because the path does not include the filename if accessing the root folder. i agree with this one `pathFile.replace("/", "")`. What do you think ?